### PR TITLE
feat(dashboard): add color picker

### PR DESCRIPTION
**Pull request by square-li**
## Overview
Add native color picker for updating threshold colors
image

### Scene Composer
For `scene-composer` package changes specifically, you can preview the component in the published storybook artifact. To do this, wait for the `Publish Storybook` action to complete below.

- Click on the workflow details
- Select the Summary item on the left
- Download the zip file

To run the storybook build locally, you need a local static web server:

```
npm install -g httpserver
cd <Extracted Zip Directory>
httpserver
```

Then open the website http://localhost:8080 to run the doc site.

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

**Comment by diehbria** [comment]
Color picker seems to be a bit overkill - has support for an alpha channel, both hex and RGBA inputs etc - while one of our main design principles is simplicity. are we sure this is the best color picker widget to go with?

File: .stylelintrc [hunk 1]
```diff
@@ -2,11 +2,13 @@
   "extends": "stylelint-config-standard",
   "rules": {
     "no-empty-source": null,
-    "selector-class-pattern": "^([a-z][a-z0-9]*)(-[a-z0-9]+)*$",
+    "selector-class-pattern": "^([a-z][a-z0-9]*)((-|_)[a-z0-9]+)*$",
```
Comment (square-li) [inline .stylelintrc:5]: This is for modifying Cloudscape-UI css classes.

File: packages/dashboard/src/components/sidePanel/sections/index.css [hunk 2]
```diff
@@ -19,6 +19,11 @@
   border-radius: 0.57rem;
 }
 
+/* remove default popover padding */
+.threshold-content-color-picker div.awsui_body_xjuzf_10pru_325.awsui_body-overflow-visible_xjuzf_10pru_339 {
```
Comment (mpopv) [inline packages/dashboard/src/components/sidePanel/sections/index.css:23]: Is there any other way we can do this? Or is this necessary? This is probably going to break on every Cloudscape version, according to what the team has said about how they autogenerate classnames

Comment (jmbuss) [inline packages/dashboard/src/components/sidePanel/sections/index.css:23]: is this a reliable selector? I used data attributes to do this in the text widget commit.

**Review by jmbuss — CHANGES_REQUESTED** [review]

File: packages/dashboard/src/components/sidePanel/sections/index.css (hunk 2 above)
Comment (square-li) [reply packages/dashboard/src/components/sidePanel/sections/index.css]: Replaced react-color lib with native color picker <input type="color">. This has been removed.

Comment (square-li) [reply packages/dashboard/src/components/sidePanel/sections/index.css]: Replaced react-color lib with native color picker <input type="color">. This has been removed.

**Comment by square-li** [comment]
> Color picker seems to be a bit overkill - has support for an alpha channel, both hex and RGBA inputs etc - while one of our main design principles is simplicity. are we sure this is the best color picker widget to go with?

Yes you are right. Switched to the native color picker `<input type="color">` instead of relying on 3rd party lib.

**Review by mpopv — APPROVED** [review]

**Review by jmbuss — APPROVED** [review]

**Pull request merged**